firmware/main.c

```diff
@@ -856,14 +862,14 @@ static void cmd_motor_linear(BaseSequentialStream *chp, int argc, char *argv[])
 	return;
 
 exit_with_usage:
-	chprintf(chp, "Usage: motor_linear -m <mode> -p [pulses] -d [direction]\r\n"
+	chprintf(chp, "Usage: motor_linear  <mode> <arguments>\r\n"
 			"\tNumber of pulses to turn (1 pulse = 1,8 degrees the motor and 0,9424mm/pulse the linear unit)\r\n"
 			"\tModes:\r\n"
-			"\t  1 - Clock/direction mode --> [require -p (number of pulses) and -d (direction of rotation)]\r\n"
-			"\t  2 - Homing\r\n"
-			"\tDirection:\r\n"
-			"\t  0 - Decrease the distance\r\n"
-			"\t  1 - Increase the distance\r\n");
+			"\t 'home' - move to reference position to calibrate distance measurement \r\n"
```

Comment:
this is much better than the above help text!
maybe merge the two 'pulses option' lines:
 `pulses <int>`, where negative values move toward x while positive move towards y.
or something like this ;)